### ISSUE TITLE
Possible to collect / display Status / Failure notices?

### ISSUE THREAD

**Issue by daytonturner** [cryptk/haomnilogic-local#72]
Im wondering if there's support in the API itself for displaying error/status messages, similar to what I can see in the Omnilogic iOS app?

I've had two status banners show up recently, a "flow sensor alert" (e.g. flow was too low, therefore pump was turned off) and a warning to "Clean T-Cell Chlorinator1"

If there was some way to know that a message like this was present (and display or alert on it from HA), instead of having to go into the omnilogic app itself to see it, that would be amazing!

**Comment by cryptk** [comment]
There is functionality for polling alarms from the controller in the underlying library, I'll use this ticket to track adding those to the integration to expose them in Home Assistant.  My kids go back to school in a couple of weeks, I should have time to work on some of these features once that happens.

**Comment by cryptk** [comment]
Unfortunately, as far as I can tell, Hayward pulled this functionality out of the local API. I'll leave this issue open for now just in case I figure out a way to get the alarm data again though.

**Comment by cryptk** [comment]
Yeah, there is no way to pull the alarm data via the Local API anymore. I have brought this up with Hayward, so hopefully they will fix this in the future, but until then, there is nothing I can do here.
